### PR TITLE
Update Qt.io.xml

### PR DESCRIPTION
**Pull request by ekaris**

### CODE REVIEW

File: src/chrome/content/rules/Qt.io.xml [hunk 1]
```diff
@@ -15,7 +15,8 @@
 	<target host="forum.qt.io" />
 	<target host="wiki.qt.io" />
 	<target host="www.qt.io" />
-
+	<target host="blog.qt.io />
```
Comment (J0WI) [inline src/chrome/content/rules/Qt.io.xml]: Missing quote here

**Comment by J0WI** [comment]
I also found:
account.qt.io
code.qt.io
login.qt.io
showroom.qt.io

**Comment by J0WI** [comment]
Could you redirect ^qt-project.org to www.qt-project.org?

**Comment by ekaris** [comment]
Its there already on line 42

**Comment by J0WI** [comment]
Ah, sorry! Thanks!

**Pull request merged**